tests/test-portals.c

```diff
@@ -173,6 +173,20 @@ global_setup (void)
   session_bus = g_bus_get_sync (G_BUS_TYPE_SESSION, NULL, &error);
   g_assert_no_error (error);
 
+  if (g_getenv("XDP_DBUS_MONITOR"))
```

Comment:
Nit: missing space before `(`

Reply:
sigh, there's always one that sneaks through. Fixed, thanks!